contrib/cirrus/git_authors_to_irc_nicks.csv

```diff
@@ -0,0 +1,8 @@
+# Comma separated mapping of author e-mail, to Freenode IRC nick.
+# When no match is found here, the username portion of the e-mail is used.
+# Sorting is done at runtime.
+# Comments (like this) and blank lines are ignored.
+
+rothberg@redhat.com,vrothberg
+santiago@redhat.com,edsantiago
+gscrivan@redhat.com,giuseppe
```

Comment:
@rhatdan above is the translation table.  Note it's checking the commit author ***E-MAIL*** address, not the github login.  So in your case, no translation is needed :smile: